components/configs/rwn-optimizations/base/3-node/master-schedulable.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: config.openshift.io/v1
+kind: Scheduler
+metadata:
+  name: cluster
+spec:
+  mastersSchedulable: true
```

Comment:
I believe this is true already if one deploys a Compact Cluster -- 3 node?
I think this might be a leftover from when we had to deploy a Full cluster -- 3 control plane, 2 workers and one wanted to specifically make the control plane scheduleable.

Reply:
Good point. Taking that one out.